src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -160,12 +160,20 @@ public async Task ConfigureAsync(CommandSettings command)
             }
 
             TaskAgent agent;
+            ISet<string> userLabels = null;
             while (true)
             {
                 runnerSettings.AgentName = command.GetRunnerName();
 
                 _term.WriteLine();
 
+                if (userLabels == null)
```

Comment:
should we also ask for labels? I don't think we ever reuse input on error.

Reply:
I feel most of the times the error is raised when there exists an agent with the same name. I feel it might be tedious for user to re-enter the labels which he already entered. But I'm fine to change the behavior.

Reply:
let's follow the existing pattern just re-ask for everything. 😄